fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownApplyProjectRule.java

```diff
@@ -57,7 +57,7 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
         newApply.getInputs().addAll(child.getInputs());
 
         ColumnRefFactory factory = context.getColumnRefFactory();
-        Map<ColumnRefOperator, ScalarOperator> allOutput = Maps.newHashMap(project.getColumnRefMap());
```

Comment:
why we can delete project

Reply:
Subquery can't output inner table columns